src/Pop/positions.js

```diff
@@ -0,0 +1,379 @@
+    { vertical: 'top', horizontal: 'right' },
+]
+
+export const CENTER_RIGHT_TOP_CENTER = [
+    { vertical: 'center', horizontal: 'right' },
+    { vertical: 'top', horizontal: 'center' },
+]
+
+export const CENTER_RIGHT_CENTER_CENTER = [
+    { vertical: 'center', horizontal: 'right' },
+    { vertical: 'center', horizontal: 'center' },
+]
+
+export const CENTER_LEFT_TOP_LEFT = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'top', horizontal: 'left' },
+]
+
+export const CENTER_LEFT_TOP_RIGHT = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'top', horizontal: 'right' },
+]
+
+export const CENTER_LEFT_TOP_CENTER = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'top', horizontal: 'center' },
+]
+
+export const CENTER_LEFT_CENTER_CENTER = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'center', horizontal: 'center' },
+]
+
+export const CENTER_CENTER_TOP_LEFT = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'top', horizontal: 'left' },
+]
+
+export const CENTER_CENTER_TOP_RIGHT = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'top', horizontal: 'right' },
+]
+
+export const CENTER_CENTER_TOP_CENTER = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'top', horizontal: 'center' },
+]
+
+export const CENTER_CENTER_CENTER_RIGHT = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'center', horizontal: 'right' },
+]
+
+export const CENTER_CENTER_CENTER_LEFT = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'center', horizontal: 'left' },
+]
+
+export const CENTER_RIGHT_BOTTOM_LEFT = [
+    { vertical: 'center', horizontal: 'right' },
+    { vertical: 'bottom', horizontal: 'left' },
+]
+
+export const CENTER_RIGHT_BOTTOM_RIGHT = [
+    { vertical: 'center', horizontal: 'right' },
+    { vertical: 'bottom', horizontal: 'right' },
+]
+
+export const CENTER_RIGHT_BOTTOM_CENTER = [
+    { vertical: 'center', horizontal: 'right' },
+    { vertical: 'bottom', horizontal: 'center' },
+]
+
+export const CENTER_LEFT_BOTTOM_LEFT = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'bottom', horizontal: 'left' },
+]
+
+export const CENTER_LEFT_BOTTOM_RIGHT = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'bottom', horizontal: 'right' },
+]
+
+export const CENTER_LEFT_BOTTOM_CENTER = [
+    { vertical: 'center', horizontal: 'left' },
+    { vertical: 'bottom', horizontal: 'center' },
+]
+
+export const CENTER_CENTER_BOTTOM_LEFT = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'bottom', horizontal: 'left' },
+]
+
+export const CENTER_CENTER_BOTTOM_RIGHT = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'bottom', horizontal: 'right' },
+]
+
+export const CENTER_CENTER_BOTTOM_CENTER = [
+    { vertical: 'center', horizontal: 'center' },
+    { vertical: 'bottom', horizontal: 'center' },
+]
```

Comment:
😱

Reply:
I'm still thinking, it should be possible to create them dynamically so we don't have three gazillion constants there... that will def. be part of the refactoring

Reply:
Nice refactor.